src/main/java/carrierManagementSystem/CarrierManagementSystemClient.java

```diff
@@ -0,0 +1,41 @@
+package carrierManagementSystem;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Client class. This class would perform the unit tests for the carrier
+ * management system.
+ */
+public class CarrierManagementSystemClient {
```

Comment:
Should this cover some error cases? For example: creating an WCDMA carrier with 4 ports, creating an LTE carrier using a WCDMA frequency, etc.

Reply:
yes, good point. 
I add some cases into the client class for the demo purpose.